### ISSUE TITLE
data is expected in fortran order

### ISSUE THREAD

**Issue by k-dominik** [ilastik/marching_cubes#19]
so, usually, one has to to `march(data.T)`. Should probably expect c-order data, or at least state it more clearly

**Comment by DerThorsten** [comment]
btw pybindd11 allows to only accept either c or fortran.
https://pybind11.readthedocs.io/en/stable/advanced/pycpp/numpy.html

this allows for csytle and will cast / convert if fstyle is given
```
(py::array_t<double, py::array::c_style | py::array::forcecast> array);
```
This only allows cstyle
```
(py::array_t<double, py::array::c_style > array);
```

This only allows fstyle
```
(py::array_t<double, py::array::c_style > array);
```
This means one can even overload on the C++ style and call different functions for c and f styled numpy arrays

**Comment by k-dominik** [comment]
cool, thanks for pointing that out!

**Comment by stuarteberg** [comment]
I ran into this today... let's remember to bump the version when this gets fixed!

**Comment by k-dominik** [comment]
related to #25

**Comment by k-dominik** [comment]
closed via #26
